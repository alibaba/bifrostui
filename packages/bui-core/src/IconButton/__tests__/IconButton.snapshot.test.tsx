import { snapshotTest } from 'testing';

describe('IconButton demo snapshot', () => {
  snapshotTest('IconButton');
});
