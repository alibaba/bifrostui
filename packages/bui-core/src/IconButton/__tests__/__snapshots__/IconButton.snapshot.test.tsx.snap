// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton demo snapshot IconButton demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-outlined bui-icon-btn-circular bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
</div>
`;

exports[`IconButton demo snapshot IconButton demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-rounded bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-square bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
</div>
`;

exports[`IconButton demo snapshot IconButton demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <button
    className="bui-icon-btn bui-icon-btn-small bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-large bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-default"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
</div>
`;

exports[`IconButton demo snapshot IconButton demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "row",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "8px",
      }
    }
  >
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-primary"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-success"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-info"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-warning"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-danger"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
  </div>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "row",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "8px",
      }
    }
  >
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-primary"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-success"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-info"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-warning"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
    <button
      className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-danger"
      type="button"
    >
      <svg
        className="bui-svg-icon"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
          }
        }
        focusable="false"
        viewBox="0 0 96 96"
      />
    </button>
  </div>
</div>
`;

exports[`IconButton demo snapshot IconButton demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-default bui-icon-btn-circular bui-icon-btn-primary bui-icon-btn-disabled"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-outlined bui-icon-btn-circular bui-icon-btn-primary bui-icon-btn-disabled"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
  <button
    className="bui-icon-btn bui-icon-btn-medium bui-icon-btn-contained bui-icon-btn-circular bui-icon-btn-primary bui-icon-btn-disabled"
    type="button"
  >
    <svg
      className="bui-svg-icon"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<path d=\\"M68.3 56.5c9.1 3.3 13.8 13.2 10.8 22.3l-.3.8C75.6 88.5 64.8 93 56.4 90l-4-1.5h-.1c-12.8-5.9-24.4-16.6-30-27.8-6.7-13.2-8.4-30.3-3.5-43.9C22.4 7.1 32.2 2.7 41.7 6.2c9.3 3.4 14 13.6 10.6 22.9-2.4 6.5-8.2 10.5-14.9 11 .8 3.8 2.1 7.6 3.9 11.2C43.4 55.4 46 59 49 62c4.7-5.7 12.1-8.1 19.3-5.5z\\"/>",
        }
      }
      focusable="false"
      viewBox="0 0 96 96"
    />
  </button>
</div>
`;
